src/amo/components/ReportAbuseButton/index.js

```diff
@@ -149,6 +150,17 @@ export class ReportAbuseButtonBase extends React.Component<InternalProps> {
     // See https://github.com/mozilla/addons-frontend/issues/4025#issuecomment-349103373
     const prompt = i18n.gettext('Report this add-on for abuse');
 
+    let reportButtonProps: Object = {
+      onClick: this.onReportButtonClick,
+    };
+
+    // When this feature flag is active, we link to the feedback form.
+    if (config.get('enableFeatureFeedbackFormLinks')) {
+      reportButtonProps = {
+        to: `/feedback/addon/${addon.slug}/`,
```

Comment:
I guess frontend doesn't have a way of referencing a route by name, etc?

Reply:
Not that I am aware of :/